### PR TITLE
Fix/route news

### PR DESCRIPTION
**Pull request by arealclimber**
#### Component
- CryptoCard
- CryptoCategory
- Footer
- NavBar
- NavBarMobile
- TickerSelectorBox
- TickerSelectorBoxMobile
- pages/trade/cfd/[instId].tsx

### CHECK LIST
-  [Naming convention](coding-convention/naming-convention.md) verification:  **checked**
-  Coding style verification: **checked**
-  new libraries: 0
-  new Class / Component: 0
-  new loop: 0
-  new recursive: 0
-  high risk operation: 0
-  SQL operation: 0

### UML
- [TBDCD00001.md](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [TBDDB00001.md](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDDB00001.md)

### CODE REVIEW

File: src/components/ticker_selector_box/ticker_selector_box.tsx [hunk 1]
```diff
@@ -138,8 +138,9 @@ const TickerSelectorBox = ({
     if (i === 0) {
       return (
         <CryptoCard
+          // inSamePage={true}
```
Comment (arealclimber) [inline src/components/ticker_selector_box/ticker_selector_box.tsx:141]: 刪掉

**Review by TzuHanLiang — COMMENTED** [review]
- LGTM

**Review by Julian0701 — APPROVED** [review]
checked

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**